tools/clang/test/HLSLFileCheck/validation/WaveSizeAttrNotAllowed3.hlsl

```diff
@@ -0,0 +1,18 @@
+// RUN: %dxc -T cs_6_6 -E main %s | FileCheck %s
+
+// this test should pass, because despite the fact that 
+// main2 isn't the current entry point function, 
+// the fact that it's callgraph-reachable from the entry
+// point function "main" gives it permission to have
+// "wavesize" as an "entry point attribute"
```

Comment:
No, this should pass because the WaveSize attribute should be ignored if it's not on a known entry point function for this compilation.  This has nothing to do with whether it's call-graph reachable.